Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "PDFXKit",
+    platforms: [
+        .iOS(.v13)
+    ],
+    products: [
+        .library(
+            name: "PDFXKit",
+            targets: ["PDFXKit"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/PSPDFKit/PSPDFKit-SP", from: "11.2.0"),
```

Comment:
Since we shipped two important security updates in 11.2.3 and 11.2.4.

```suggestion
        .package(url: "https://github.com/PSPDFKit/PSPDFKit-SP", from: "11.2.4"),
```